app/views/devices_guidance/how_to_order_for_social_care.html.erb

```diff
@@ -0,0 +1,215 @@
+<% content_for :devices_service, true %>
+<% content_for :title, 'Order laptops and get internet access for care leavers' %>
+
+<%- content_for :before_content do %>
+  <% breadcrumbs([{ "Home" => root_path },
+                  "Get laptops",
+                 ]) %>
+<% end %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Order laptops and get internet access for care leavers
+    </h1>
+    <p class="govuk-body">
+      This guidance explains which care leavers are eligible to receive laptops, and how local authorities (LAs) can place
+      orders through the Department for Education’s Get help with technology service.
+    </p>
+
+    <p class="govuk-body">
+      Care leavers cannot order directly.
+    </p>
+
+    <p class="govuk-body">
+      We recommend that you get set up to order as soon as possible to make sure you receive your preferred choice of laptops.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      Which care leavers are eligible for this support
+    </h2>
+
+    <p class="govuk-body">
+      DfE is providing these to support care leavers who:
```

Comment:
Please change to: DfE is providing these to support care leavers your LA is responsible for, who: